NumLib/TimeStepping/Algorithms/CreateEvolutionaryPIDcontroller.cpp

```diff
@@ -41,22 +42,18 @@ std::unique_ptr<TimeStepAlgorithm> createEvolutionaryPIDcontroller(
     //! \ogs_file_param{prj__time_loop__time_stepping__EvolutionaryPIDcontroller__rel_dt_max}
     auto const rel_h_max = config.getConfigParameter<double>("rel_dt_max");
 
-    auto specific_times_opt =
+    auto specific_times =
         //! \ogs_file_param{prj__time_loop__time_stepping__EvolutionaryPIDcontroller__specific_times}
-        config.getConfigParameterOptional<std::vector<double>>(
-            "specific_times");
-    std::vector<double> dummy_vector;
-    std::vector<double>& specific_times =
-        (specific_times_opt) ? *specific_times_opt : dummy_vector;
-    if (specific_times.size() > 0)
+        config.getConfigParameter<std::vector<double>>(
```

Comment:
This parameter is optional by calling getConfigParameter(string, default). However, the problem is how to  document this parameter as an optional one.